src/DIG/Dialog.jsx

```diff
@@ -161,6 +161,8 @@ confirmButton = confirmButton ?? (
   />
 );
 
+if (!forwardedProps?.open) return <></>;
```

Comment:
I decided to add this condition because without it, this component would render many times on the activity page without even being called